paths_cli/tests/test_utils.py

```diff
@@ -0,0 +1,39 @@
+from paths_cli.utils import *
+
+class TestOrderedSet:
+    def setup(self):
+        self.set = OrderedSet(['a', 'b', 'a', 'c', 'd', 'c', 'd'])
+
+    def test_len(self):
+        assert len(self.set) == 4
+
+    def test_empty(self):
+        ordered = OrderedSet()
+        assert len(ordered) == 0
+        for _ in ordered:
+            raise RuntimeError("This should not happen")
+
+    def test_order(self):
+        for truth, beauty in zip("abcd", self.set):
+            assert truth == beauty
+
+    def test_add_existing(self):
+        assert len(self.set) == 4
+        self.set.add('a')
+        assert len(self.set) == 4
+        assert list(self.set) == ['a', 'b', 'c', 'd']
+
+    def test_contains(self):
+        assert 'a' in self.set
+        assert not 'q' in self.set
```

Comment:
could I also ask for a 
```suggestion
        assert not 'q' in self.set
        assert 'q' not in self.set
```